libs/ActivityKit/source/activities/ShapeRecognition.cpp

```diff
@@ -0,0 +1,53 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+// LCOV_EXCL_START
+
+#include "ShapeRecognition.h"
+
+namespace leka::activity {
+
+void ShapeRecognition::start()
+{
+	_score = 0;
+	_shape = {};
+
+	_backup_callback = _rfidkit.getCallback();
+	std::shuffle(shape_span.begin(), shape_span.end(), std::mt19937(time(nullptr)));
```

Comment:
I don't think you need a std::span here, you can just pass the std::array directly, no?